sdcm/utils/common.py

```diff
@@ -340,7 +340,7 @@ def __init__(self, objects, timeout=6, num_workers=None, disable_logging=False):
         self.num_workers = num_workers
         self.disable_logging = disable_logging
 
-    def run(self, func):
+    def run(self, func, ignore_exception=False):
```

Comment:
please add a docstring to explain that is the caller responsibility to handle None in case of ignored exceptions

Reply:
I would call it `ignore_exceptions` (plural) since it can be one or more

Reply:
fixed